ompi/communicator/ft/comm_ft.c

```diff
@@ -132,15 +134,21 @@ int ompi_comm_shrink_internal(ompi_communicator_t* comm, ompi_communicator_t** n
     OPAL_OUTPUT_VERBOSE((5, ompi_ftmpi_output_handle,
                          "%s ompi: comm_shrink: Agreement on failed processes",
                          OMPI_NAME_PRINT(OMPI_PROC_MY_NAME) ));
+#if OPAL_ENABLE_DEBUG
     start = PMPI_Wtime();
+#endif
     opal_mutex_lock(&ompi_group_afp_mutex);
     ompi_group_intersection(comm->c_remote_group, ompi_group_all_failed_procs, &failed_group);
     opal_mutex_unlock(&ompi_group_afp_mutex);
+#if OPAL_ENABLE_DEBUG
     stop = PMPI_Wtime();
+#endif
     OPAL_OUTPUT_VERBOSE((10, ompi_ftmpi_output_handle,
                          "%s ompi: comm_shrink: group_inter: %g seconds",
                          OMPI_NAME_PRINT(OMPI_PROC_MY_NAME), stop-start));
```

Comment:
I think we also need `#if OPAL_ENABLE_DEBUG` guards around the `OPAL_OUTPUT_VERBSE()` call which prints `stop-start`?